src/ucs/stats/stats.c

```diff
@@ -460,6 +465,138 @@ void ucs_stats_node_free(ucs_stats_node_t *node)
     }
 }
 
+static inline uint32_t ucs_stats_class_id_to_aggregate_sum_base_offset_get(
+        const char *class_name, unsigned *class_id,
+        size_t num_counters_in_class)
+{
+    uint32_t offset = 0;
+
+    if (ucs_likely(*class_id != 0)) {
+        offset = g_stats_class_id_to_name[*class_id].aggregate_sum_class_offset;
```

Comment:
if (*class_id==0) {
  return ...
}
no need for else

Reply:
@yosefe , Well, there's code in the else - This code assigns an offset for an unknown class_id - It's required.
Could you please explain the comment?